recipes/default.rb

```diff
@@ -15,14 +15,14 @@
   node['rackspace_users']['data_bag'],
   node['rackspace_users']['data_bag_item']).to_hash.select { |user, user_data| user != 'id' }
```

Comment:
It might be worth putting the users under their own `users` key rather than in the root of the data bag so that they have their own namespace.  The logic is cleaner and doesn't fail in edge cases like creating a user under the name of `id`.